docs/setup.md

```diff
@@ -51,16 +51,27 @@ where `${CUDA_VERSION}` can be `80` (8.0), `90` (9.0), `92` (9.2), or `100` (10.
 
 Developers will be better served by pointing `$PYTHONPATH` to the root of the git-cloned source.
 
-### → on AWS...
+### → on AWS and Azure ...
```

Comment:
As this really mainly about Anaconda now and not about either Azure or AWS: How about we call this section:
```
### in an Anaconda environment
```
and then 
```
In an Anaconda environment such as the one provided by the [AWS DeepLearning AMI](https://aws.amazon.com/amazon-ai/amis/) users only need to run the following line to install sockeye (on an instance without a GPU):
```

Reply:
I like this. Shall I make this change and re-submit?

Reply:
I'd use the `CUDA_VERSION` environment variable and add a sentence as done in the pip section:
```
where ${CUDA_VERSION} can be 80 (8.0), 90 (9.0), 92 (9.2), or 100 (10.0).
```

Or maybe just once outside the pip and anaconda sections mention that `CUDA_VERSION` in the following refers to the cuda version present on the system.